tools/test.py

```diff
@@ -570,11 +571,17 @@ def PrintError(str):
 
 
 def CheckedUnlink(name):
-  try:
-    os.unlink(name)
-  except OSError, e:
-    PrintError("os.unlink() " + str(e))
-
+  while True:
+    try:
+      os.unlink(name)
+    except OSError, e:
+      # On Windows unlink() fails if another process has the file open.
+      # Yield and try again, and it'll succeed.
+      if os.name == 'nt' and e.errno == errno.EACCES:
```

Comment:
`sys.platform == 'win32'` is slightly more idiomatic.  Maybe the comment could mention that 'another process' means a virus scanner or an indexer because the only two calls to CheckedUnlink() are for temp files.  An unwitting reader could easily assume that the loop is not in fact necessary.